src/plugins/flow.js

```diff
@@ -450,24 +450,34 @@ pp.flowParseObjectType = function (allowStatic, allowExact) {
       }
       nodeStart.callProperties.push(this.flowParseObjectTypeCallProperty(node, isStatic));
     } else {
-      propertyKey = this.flowParseObjectPropertyKey();
-      if (this.isRelational("<") || this.match(tt.parenL)) {
-        // This is a method property
-        if (variance) {
-          this.unexpected(variance.start);
+      if (this.match(tt.ellipsis)) {
+        if (!allowSpread) {
+          this.unexpected();
         }
-        nodeStart.properties.push(this.flowParseObjectTypeMethod(startPos, startLoc, isStatic, propertyKey));
+        this.expect(tt.ellipsis);
+        node.argument = this.flowParsePrimaryType();
```

Comment:
Should this be `this.flowParseType()`? That seems to be what the [Flow parser is doing](https://github.com/facebook/flow/blob/b33ce6ea80a426e8e971ab08a5bc3a107fd9ab33/src/parser/type_parser.ml#L519)